zetaclient/chains/evm/signer/outbound_tracker_reporter.go

```diff
@@ -68,6 +69,15 @@ func (signer *Signer) reportToOutboundTracker(
 				continue
 			}
 
+			// stop if the cctx is already finalized
```

Comment:
I saw a lots unnecessary outbound tracker submissions in E2E tests. I tracked down the root cause: 

the outbound ballot finalizes so fast in local network and the tracker reporter is still submitting tracker without checking the CCTX's status. This will not happen on mainnet due to the slow finality.